src/plugins/share/server/plugin.ts

```diff
@@ -37,6 +37,9 @@ export class SharePlugin implements Plugin<SharePluginSetup, SharePluginStart> {
       getUrl: async () => {
         throw new Error('Locator .getUrl() currently is not supported on the server.');
       },
+      getRedirectUrl: async () => {
+        throw new Error('Locator .getRedirectUrl() currently is not supported on the server.');
```

Comment:
can we implement this on the server as well, reporting will want to generate redirect url on the server when capturing pdf/png reports.